packages/patternfly-4/react-core/src/components/Progress/Progress.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/Progress/styles.css';
+import { css, getModifier } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import ProgressContainer, { ProgressMeasureLocation, ProgressStatus } from './ProgressContainer';
+
+export { ProgressMeasureLocation, ProgressStatus } from './ProgressContainer';
```

Comment:
Not needed, but it might be better to export these in the barrel file from their files instead of this one. It makes following exports easier.

Reply:
Sure thing!